src/macaron/slsa_analyzer/checks/check_result.py

```diff
@@ -37,7 +110,36 @@ class Confidence(float, Enum):
     MEDIUM = 0.7
 
     #: A low confidence score.
-    LOW = 0.5
+    LOW = 0.4
+
+    @classmethod
+    def normalize(cls, evidence_weight_map: EvidenceWeightMap) -> "Confidence":
+        """Normalize the score based on the provided evidence weight map.
+
+        The values in the evidence weight map are expected to be positive. Expect invalid results if negative weights
+        are passed to this function.
+
+        Parameters
+        ----------
+        evidence_weight_map: EvidenceWeightMap
+            The map that contains the detected evidence and their corresponding weight.
+
+        """
+        max_score = evidence_weight_map.get_max_score()
+        # If the maximum score is zero, there is no need to normalize and just return the highest confidence level.
+        if max_score == 0:
+            return cls.HIGH
+
+        score = evidence_weight_map.get_score()
+
+        # If the difference is zero, there is no need to normalize.
+        if cls.HIGH - cls.LOW == 0:
+            return cls.HIGH
```

Comment:
I believe `cls.HIGH` and `cls.LOW` are not modifiable by the caller of this function right :thinking: ? 
I wonder if this if statement will be True unless the developer explicitly modify the float values of the enum listed in `Confidence`.

Reply:
The constant scores cannot be modified at runtime. This check prevents issues in case these values are equal in the future.